src/database.ts

```diff
@@ -1879,28 +1895,34 @@ class Database extends common.GrpcServiceObject {
    */
   restore(
     backupName: string,
-    optionsOrCallback?: CallOptions | RestoreDatabaseCallback,
+    optionsOrCallback?: RestoreOptions | RestoreDatabaseCallback,
```

Comment:
Just calling this out to make sure the PR is merged as a breaking change.

```diff
-database.restore({ timeout: 0 });
+database.restore({ gaxOptions: { timeout: 0 } });
```

Reply:
Oh my bad. I was meant to send out a separate PR for this to group all the breaking changes together (e.g. #1254). I will then rebase this PR so that this explicitly doesn't need to be marked as a breaking change.

Reply:
Sent out #1278. Will rebase once we're able to merge that in together with all the other breaking changes (likely in early January).

Reply:
The breaking change is no longer needed.